src/Microsoft.ApiDesignGuidelines.Analyzers/UnitTests/PropertiesShouldNotReturnArraysTests.cs

```diff
@@ -19,5 +19,56 @@ protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer()
         {
             return new CSharpPropertiesShouldNotReturnArraysAnalyzer();
         }
+
+        [Fact]
+        public void TestCSharpPropertiesShouldNotReturnArraysArray()
```

Comment:
bad naming,  the first 'Arrays' is part of the rule name, last 'Array' was trying to tell this test case is test return array and should trigger diagnostic. 
Usually what's the better convention here?